components/board/jetson/data.go

```diff
@@ -17,15 +17,17 @@ const (
 	jetsonOrinNano = "jetson_orin_nano"
 )
 
+//TODO: fix ngpio numbers in pin definitions for the jetsonTX1, jetsonNano
```

Comment:
Make a Jira ticket and then edit this to be `TODO [RSDK-1234] fix ngpio...`. That way, we won't forget about this.

Reply:
is it worth making a jira ticket if these boards aren't supported? there's no point in finding correct definitions unless we're able to test the board

Reply:
Yeah, otherwise this is going to get lost. We can mark it as low priority and get to it 3 years from now, but the rule is that every TODO needs a Jira ticket. (There might be todos in the code from before this rule, but we shouldn't add more of them.)